example/lib/main.dart

```diff
@@ -0,0 +1,252 @@
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Chat Ui Demo',
+      theme: ThemeData(primarySwatch: Colors.blue),
+      home: const MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({Key? key}) : super(key: key);
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  final _scrollController = ScrollController();
+  bool _isLastPage = false;
+  final _backGroundImage =
+      "https://images.pexels.com/photos/1591447/pexels-photo-1591447.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500";
+  final _list = [
+    Message(
+      id: '1',
+      message: "Hi",
+      createdAt: DateTime.now(),
+      sendBy: "Dhvanit",
+    ),
+    Message(
+      id: '2',
+      message: "Hi",
+      createdAt: DateTime.now(),
+      sendBy: "Faiyaz",
+    ),
+    Message(
+      id: '3',
+      message: "How are you?",
+      createdAt: DateTime.now(),
+      sendBy: "Dhvanit",
+    ),
+    Message(
+      id: '4',
+      message: "I am fine",
+      createdAt: DateTime.now(),
+      sendBy: "Faiyaz",
+    ),
+    Message(
+      id: '5',
+      message: "Okay",
+      createdAt: DateTime.now(),
+      sendBy: "Dhvanit",
+      reaction: '❤️',
+    ),
+    Message(
+      id: '6',
+      message: "Do you have idea about this?",
+      createdAt: DateTime.now(),
+      sendBy: "Faiyaz",
+    ),
+    Message(
+      id: '7',
+      message:
+          "https://docs.flutter.dev/development/packages-and-plugins/developing-packages",
+      createdAt: DateTime.now(),
+      sendBy: "Faiyaz",
+      replyMessage: ReplyMessage(
+        message: "Do you have idea about this?",
+        replyTo: "Dhvanit",
+        replyBy: "Faiyaz",
+      ),
+    ),
+    Message(
+      id: '8',
+      message:
+          "https://images.pexels.com/photos/586744/pexels-photo-586744.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500",
+      createdAt: DateTime.now(),
+      sendBy: "Dhvanit",
+    ),
+    Message(
+      id: '9',
+      message:
+          "https://images.pexels.com/photos/586744/pexels-photo-586744.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500",
+      createdAt: DateTime.now(),
+      sendBy: "Faiyaz",
+      reaction: '❤️',
+    ),
+    Message(
+      id: '6',
+      message: "EveryThing is fine?",
+      createdAt: DateTime.now(),
+      sendBy: "Faiyaz",
+    ),
+    Message(
+      id: '6',
+      message: "😂😂",
+      createdAt: DateTime.now(),
+      sendBy: "Dhvanit",
+    ),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Flutter Chat Ui Demo')),
+      body: ChatView(
+        scrollController: _scrollController,
+        messageList: _list,
+        currentUser: "Dhvanit",
+        messageListConfiguration: MessageListConfiguration(
+          height: 500,
```

Comment:
what is this height does ?

Reply:
User can set chat UI in screen according to their preference.